tests/unittests/tests-gcoap/Makefile.include

```diff
@@ -0,0 +1,8 @@
+USEPKG += nanocoap
+USEMODULE += gnrc_sock_udp
+
+USEMODULE += gcoap
+USEMODULE += gnrc_udp
+USEMODULE += gnrc_ipv6
```

Comment:
Could you maybe make this more clear (e.g. by adding "uncomment if needed" like in the [`gnrc_minimal`](https://github.com/RIOT-OS/RIOT/blob/master/examples/gnrc_minimal/Makefile#L12) example)?